# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/moz.build
# Commit: fa57574775b2
# Full Hash: fa57574775b29f4a03667d9c350a1be269adfaf7
# Author: Dragana Damjanovic <dd.mozilla@gmail.com>
# Date: 2019-09-26 21:35:42
# Regressor Bug: 1577643
# File Overlap Count: 1
# Description:
#   Bug 1577643 - Implement a security info class for the quic transport. r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44073
# ==============================================================================

diff -r 02f72f7040e4 -r fa57574775b2 netwerk/protocol/http/moz.build
--- a/netwerk/protocol/http/moz.build	Thu Sep 26 10:14:53 2019 +0000
+++ b/netwerk/protocol/http/moz.build	Wed Sep 25 20:23:56 2019 +0000
@@ -108,6 +108,7 @@
     'NullHttpChannel.cpp',
     'NullHttpTransaction.cpp',
     'ParentChannelListener.cpp',
+    'QuicSocketControl.cpp',
     'TunnelUtils.cpp',
 ]
 